Modules/ThirdParty/HDF5/src/itkhdf5/config/cmake/HDF518_Examples.cmake.in

```diff
@@ -47,7 +47,7 @@ set(ADD_BUILD_OPTIONS "${ADD_BUILD_OPTIONS} -DCTEST_CONFIGURATION_TYPE:STRING=${
 ##################################################################
 
 if(NOT DEFINED INSTALLDIR)
-  set(INSTALLDIR "@CMAKE_INSTALL_PREFIX@")
+  set(INSTALLDIR "C:/Program Files/HDF_Group/@HDF5_PACKAGE_NAME@/@HDF5_PACKAGE_VERSION@")
```

Comment:
I don't think this affect ITK, and I think this is a bug in upstream.

Hard coded absolute paths are bad.

Reply:
:+1: